packages/client/src/components/app/GridBlock.svelte

```diff
@@ -35,6 +35,7 @@
 
   let grid
 
+  $: isDark = $component.isDark
```

Comment:
We can use the new context technique to get the theme like the signature field does.